python/tvm/driver/tvmc/autotuner.py

```diff
@@ -133,6 +127,45 @@ def add_tune_parser(subparsers):
         default=None,
         help="change the data layout of the whole graph",
     )
+    parser.add_argument(
+        "--enable-autoscheduler",
+        help="enable tuning the graph through the autoscheduler",
+        action="store_true",
+    )
+
+    auto_scheduler_group = parser.add_argument_group(
+        "Autoscheduler options",
+        "Autoscheduler options, used when --enabled-auto-scheduler is provided",
+    )
+
+    auto_scheduler_group.add_argument(
+        "--cache-line-bytes",
+        type=int,
+        default=64,
+        help="the size of cache line in bytes",
+    )
+    auto_scheduler_group.add_argument(
+        "--num-cores",
+        type=int,
+        default=4,
+        help="the number of device cores",
+    )
+    auto_scheduler_group.add_argument(
+        "--vector-unit-bytes",
+        type=int,
+        default=16,
+        help="the width of vector units in bytes",
+    )
+    auto_tuning_group = parser.add_argument_group(
+        "Autotuning options",
+        "Autotuning options, used when the autoscheduler is not enabled",
+    )
```

Comment:
Since this is also for me to familiarize with the Autoscheduler, I would prefer to add here anything that you and @merrymercy think relevant in terms of Autoscheduler options.